index.html

```diff
@@ -1615,6 +1618,179 @@ <h3>
+          title="https://example.com/manifest.webmanifest">
+          {
+            "id": "https://example.com/app",
+            "name": "My App",
+            "icons": [{
+              "src": "icon/hd_hi",
+              "sizes": "128x128"
+            }],
+            "start_url": "/app/index.html",
+            "scope": "/app",
+            "display": "standalone",
+            "scope_extensions": [
+              { "type": "origin", "value": "https://example.co.uk" },
+              { "type": "origin", "value": "https://help.example.com" }]
+          }
+        </pre>
+        <p>
+          The following shows 2 [=web-app-origin-association=] files which can
+          be downloaded from the `.well-known` path of the [=origins=]  listed 
+          in the `scope_extensions` member.
+        </p>
+        <pre class="example json" 
+        title="https://example.co.uk/.well-known/web-app-origin-association">
+          {
+            "https://example.com/app": {
+              "scope": "/app"   
+            }
+          }
+        </pre>
+        <pre class="example json" 
+        title="https://help.example.com/.well-known/web-app-origin-association">
+          {
+            "https://example.com/app": {
+              "scope": "/"   
+            }
+          }
+        </pre>
+        <p> 
+          The navigation scope of this app should be all URLs that are 
+          [=manifest/within scope=] of each of 
+          `https://example.com/app`, `https://example.co.uk/app`, and 
+          `https://help.example.com`. 
+        </p>
+      </section>
+      <section>
+      <h2>Processing the scope_extensions member</h2>
+      <p>
+        To <dfn>process the `scope_extensions` member</dfn>, given
+        [=ordered map=] |json:ordered map| and [=ordered map=]
+        |manifest:ordered map|:
+      </p>
+      <ol class="algorithm">
+        <li>Let |processedScopeExtensions:list| be a new [=list=].
+        </li>
+        <li>Set |manifest|["scope_extensions"] to |processedScopeExtensions|.
+        </li>
+        <li>If |json|["scope_extensions"] doesn't [=map/exist=] or
+          |json|["scope_extensions"] is not a [=list=], return.
+        </li>
+        <li>[=list/For each=] <var>entry</var> of |json|["scope_extensions"]:
+          <ol>
+            <li>If <var>entry</var>["type"] or <var>entry</var>["value"]
+              is undefined, [=iteration/continue=].
+            </li>
+            <li>If |entry|["type"] is not `origin`, 
+              [=iteration/continue=].
+            </li>
+            <li>If <var>entry</var>["value"] is not a valid [=URL=], 
+              [=iteration/continue=].
+            </li>
+            <li>Set <var>entry</var>["value"] to its own [=origin=].
+            </li>
+            <li>Validate |entry| using the steps from [=validate scope 
+              extensions=], given [=ordered map=] |entry:ordered map|, and 
+              [=URL=] |resolved_scope:URL|.
```

Comment:
we also need to call the algorithm with our manifest_id?

Reply:
Added manifest_id to params.